change.py

```diff
@@ -1,3 +1,24 @@
-def make_change(amount:str) -> dict:
+"""Containing function make_change"""
+def make_change(amount: str) -> dict:
+    """
+    Returns dict that contains change of
+    given amount of money using minimum coins.
+    """
+    denominations = [2, 1, 0.5, 0.25, 0.1, 0.05]
+    money = float(amount[:amount.index(' ')])
+    change = {1: 0, 2: 0, 0.5: 0, 0.25: 0, 0.1: 0, 0.05: 0}
+    change_res = {1: 0, 2: 0, 50: 0, 25: 0, 10: 0, 5: 0}
 
-    return {}
\ No newline at end of file
+    repl = [(1, 1), (2, 2), (0.5, 50), (0.25, 25), (0.1, 10), (0.05, 5)]
```

Comment:
! Try to avoid duplication in these variables: denominations, change, change_res, repl

Reply:
I've made corrections as you said.
Can you see the changes, or do I need to do another Pull request?